src/api-service/__app__/onefuzzlib/workers/scalesets.py

```diff
@@ -936,6 +946,75 @@ def set_state(self, state: ScalesetState) -> None:
                 )
             )
 
+    def sync_auto_scale_settings(self) -> Optional[Error]:
+        from .pools import Pool
+
+        # No need to update tables when in shutdown state
+        if self.state == ScalesetState.shutdown:
+            return
+
+        logging.info(
+            "Trying to sync auto scale settings for scaleset %s" % self.scaleset_id
+        )
+
+        auto_scale_profile = get_auto_scale_profile(self.scaleset_id)
+        if auto_scale_profile is None:
+            auto_scale_profile_failed = Error(
+                code=ErrorCode.UNABLE_TO_FIND,
+                errors=[
+                    "Failed to get auto scale profile for scaleset %s"
+                    % self.scaleset_id
+                ],
+            )
+            logging.error(auto_scale_profile_failed)
+            return auto_scale_profile_failed
+
+        minimum = auto_scale_profile.capacity.minimum
+        maximum = auto_scale_profile.capacity.maximum
+        default = auto_scale_profile.capacity.default
+
+        scale_out_amount = 1
+        scale_out_cooldown = 10
+        scale_in_amount = 1
+        scale_in_cooldown = 15
+
+        for rule in auto_scale_profile.rules:
+            scale_action = rule.scale_action
+            if scale_action.direction == "Increase":
+                scale_out_amount = scale_action.value
+                logging.info("Number of seconds: %d" % scale_out_cooldown)
```

Comment:
The cooldowns (at least as they are defined on line 977/979) are in minutes. It's possible the response from the azure API is in seconds though.

Reply:
The response is in seconds.